friend/friends.js

```diff
@@ -0,0 +1,52 @@
+const mongoose = require('mongoose');
+
+// mongoose.connect('mongodb://localhost/friends')
+
+var db = mongoose.connection;
+
+db.on('error', console.error.bind(console, 'connection error:'))
+db.once('open', function() {
+    console.log('connected to friends database')
+});
+
+const definition = {
+    firstName: {
+        type: String,
+        required: true
+    },
+    lastName: {
+        type: String,
+        required: true
+    },
+    age: {
+        type: Number,
+        required: true,
+        min: 1,
+        max: 120
+    },
+    createOn: {
```

Comment:
Since we have the `timestamps: true` below, we don't need `createOn`. They pretty much do the same thing.